src/app/components/home/home.component.ts

```diff
@@ -1,8 +1,22 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
 
 @Component({
   selector: 'app-home',
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.scss'],
 })
-export class HomeComponent {}
+export class HomeComponent implements OnInit {
+  constructor(private router: Router) {}
+
+  public redirect() {
+    if (localStorage.getItem('redirectUrl') !== null) {
+      this.router.navigateByUrl(`${localStorage.getItem('redirectUrl')}`);
+      localStorage.clear();
```

Comment:
We should probably just clear the 'redirectUrl`, this could cause problems if local storage was used from anything else.

Reply:
I've replaced localstorage.clear() with localstorage.removeItem('redirectUrl')